lighthouse-cli/test/fixtures/oopif/oopif-outer.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+
+<h1>Hello oopif</h1>
+<script>
+  // We construct the iframe in JS only to synchronously set the location hash
+  const iframe = document.createElement('iframe');
+  iframe.id='oopif';
+  // Experimental hack to detect we're OOPIF clientside.
+  // The deviceMemory API seems to not work for OOPIF, so we match values between outer/inner frames
```

Comment:
> shall we assert something about this? like adding an audit failure when it's not oopif'd and adding to expectations so we know we're not testing what we think we're testing anymore.

ooh, good idea. Would throwing an error in the page (`throw new Error('iframe not out of process, see #11921')`) and asserting `errors-in-console` is empty work? I guess one issue is it would probably have to be thrown in `outer` not `inner` to be caught by the console gatherer.

Reply:
yeah i like asserting this.  but i also don't feel super confident about this devicememory hack.

ideally i could assert against the `pid`s in the trace.  but there's no easy way to do that in smokehouse. :/

Reply:
custom audits should work in smokehouse... 🔥 🐺 🔥

Reply:
> custom audits should work fine in smokehouse...

(making sure a custom audit is included in the bundled version is slightly tricky if specified with a path in the config, but rather than messing with the bundling step to get it in there, if the oopif config directly `require`s the audit and then it's included via `audits: [{implementation: CustomAudit}]` instead of with a path, it should be found and bundled just fine)